x-pack/plugins/security_solution/public/management/cypress/tasks/alerts.ts

```diff
@@ -48,7 +48,7 @@ export const waitForEndpointAlerts = (
           return (streamedAlerts.hits.total as estypes.SearchTotalHits).value > 0;
         });
       },
-      { timeout }
+      { timeout, interval: 2000 }
```

Comment:
Interesting why `interval` is either `1000` or `2000` and not `2000` all the time? Ideally it should be in a constant with a verbose name as magic constants cause more questions.